src/cf-pagination/src/cf-pagination.less

```diff
@@ -8,15 +8,13 @@
 //
 
 // Color variables
-// `$color-` variables referenced in comments are from 18F's U.S. Web Design Standards
-// https://github.com/18F/web-design-standards/blob/staging/src/stylesheets/core/_variables.scss)
 
-@pagination-text-color:       #323a45; // $color-gray-dark
-@pagination-bg-color:         #f1f1f1; // $color-gray-lightest
+@pagination-text:               @gray;
+@pagination-bg:                 @gray-10;
```

Comment:
TRIVIAL: These can be indented back a few spaces to the left and then the `@pagination-btn-min-width-px:` doesn't need the extra spaces.